docs/process.md

```diff
@@ -84,12 +91,16 @@ What should reviewers be checking for?
 
 ### Stage 4: Social media
 
-Once published, the writer creates social media posts and shares links on the “DCR Social Media” channel on Matrix. This gives the wider community an opportunity to engage with the article, adding comments, likes and retweets etc.
+Once published, the writer creates social media posts and shares links on the “DCR Social Media” and "DCR Writers" channels on Matrix. This gives the wider community an opportunity to engage with the article, adding comments, likes and retweets etc.
```

Comment:
Is it necessary to post SM links in both #media and #writers?

Reply:
Not sure, would it get more views doing both or do you think it doesn’t  make much difference?

Reply:
I think posting in #writers doesn't add anything, but adds more boring routine to the writers, making the work less fun.

I suggest to evaluate if each extra step required from the authors is adding value, and try to minimize them.

Posting links in #media is nice to have a stream of content flowing there. People lurking in chats may add a useful comment or idea. It *may* trigger a few more likes/engangement, but don't expect much. Few people are active in chats and many of them will discover the tweet regardless of the chat.

Reply:
Done 👍